### ISSUE TITLE
Chess Option Missing

### ISSUE THREAD

**Issue by Cosmic-Warfighter** [Monika-After-Story/MonikaModDev#3091]
Monika brought up chess, and I played multiple games with her, always accepting the rematch, but after I declined a rematch and returning to the main part of the game, the option to play chess with her doesn't appear in the play menu

**Comment by jmwall24** [comment]
First thing I have to ask...have you ever cheated at chess?

**Comment by Cosmic-Warfighter** [comment]
No. I didnt even know you could cheat until I started looking up a solution
to the problem online.

**Comment by jmwall24** [comment]
Can you send your `persistent`, please?  If you are unsure where it is located, please look [here](https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#i-need-to-restore-an-older-persistent)

**Comment by Cosmic-Warfighter** [comment]
sorry the email version cut out the part about changing the file type. I just now figured out i didn't respond properly.

[persistent.txt](https://github.com/Monika-After-Story/MonikaModDev/files/2611907/persistent.txt)

**Comment by Cosmic-Warfighter** [comment]
update- I downloaded the dev build and the chess option still isn't available

**Comment by jmwall24** [comment]
@Cosmic-Warfighter  put this rpyc in your game folder, it'll fix it for you.

[zz_dropfix.zip](https://github.com/Monika-After-Story/MonikaModDev/files/2613610/zz_dropfix.zip)
